models/pose_estimation_mediapipe/demo.cpp

```diff
@@ -0,0 +1,2913 @@
+        // binarize mask
+        threshold(invertRotationMask, invertRotationMask, 1, 255, THRESH_BINARY);
+
+        /* 2*2 person bbox: [[x1, y1], [x2, y2]]
+        # 39*5 screen landmarks: 33 keypoints and 6 auxiliary points with [x, y, z, visibility, presence], z value is relative to HIP
+        # Visibility is probability that a keypoint is located within the frame and not occluded by another bigger body part or another object
+        # Presence is probability that a keypoint is located within the frame
+        # 39*3 world landmarks: 33 keypoints and 6 auxiliary points with [x, y, z] 3D metric x, y, z coordinate
+        # img_height*img_width mask: gray mask, where 255 indicates the full body of a person and 0 means background
+        # 64*64*39 heatmap: currently only used for refining landmarks, requires sigmod processing before use
+        # conf: confidence of prediction*/
+        return tuple<Mat , Mat, Mat, Mat, Mat, float>(bbox, landmarks, rotatedLandmarksWorld, invertRotationMask, heatmap, valConf);
+    }
+};
+
+std::string keys =
+"{ help  h          |                                               | Print help message. }"
+"{ model m          | person_detection_mediapipe_2023mar.onnx       | Usage: Path to the model, defaults to person_detection_mediapipe_2023mar.onnx  }"
+"{ input i          |                                               | Path to input image or video file. Skip this argument to capture frames from a camera.}"
+"{ conf_threshold  | 0.5                                            | Usage: Filter out hands of confidence < conf_threshold. }"
+// "{ score_threshold  | 0.5                                           | Usage: Set the minimum needed confidence for the model to identify a person, defaults to 0.5. Smaller values may result in faster detection, but will limit accuracy. Filter out persons of confidence < conf_threshold. }"
+// "{ nms_threshold    | 0.3                                           | Usage: Suppress bounding boxes of iou >= nms_threshold. Default = 0.3. }"
+"{ top_k            | 1                                             | Usage: Keep top_k bounding boxes before NMS. }"
+"{ save s           | true                                          | Usage: Specify to save file with results (i.e. bounding box, confidence level). Invalid in case of camera input. }"
+"{ vis v            | true                                          | Usage: Specify to open a new window to show results. Invalid in case of camera input. }"
+"{ backend bt       | 0                                             | Choose one of computation backends: "
+"0: (default) OpenCV implementation + CPU, "
+"1: CUDA + GPU (CUDA), "
+"2: CUDA + GPU (CUDA FP16), "
+"3: TIM-VX + NPU, "
+"4: CANN + NPU}";
+
+Mat visualizePerson(Mat img, Mat results, double fps = -1)
+{
+    Mat resImg = img.clone();
+    if (fps > 0)
+        putText(resImg, format("FPS: %2f", fps), Point(0, 15), FONT_HERSHEY_SIMPLEX, 0.5, Scalar(0, 0, 255));
+
```

Comment:
fps option should be remove from `visualizePerson` to `visualize` function.

Reply:
Maybe we should delete `visualizePerson` function.

Reply:
Ok delete